### PR TITLE
Update test data for Mongo, CLI, etc.

### PR DESCRIPTION
**Pull request by NevilleS**
### Code Changes

* [X] Update test Mongo dataset & test env TOML

### Steps to Confirm

* [X] Complete release checklist

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### Description Of Changes

This is a few local changes I made while completing the release checklist here: #2948.

### CODE REVIEW

**Comment by NevilleS** [comment]
I think this will fail a few unit/integration tests, so kicking off a CI job to check...

**Comment by NevilleS** [comment]
OK, this failed some Mongo integration tests (not surprisingly). I hadn't dug into the integration tests yet.

**Comment by pattisdr** [comment]
yes undoing the integration test changes here https://github.com/ethyca/fides/pull/2785/files

File: src/fides/data/sample_project/sample_resources/mongo_example_test_dataset.yml [hunk 1]
```diff
@@ -266,24 +266,24 @@ dataset:
...
+                fides_meta:
                   data_type: string
                   references:
                     - dataset: mongo_test
                       field: internal_customer_profile.customer_identifiers.derived_phone
                       direction: from
               - name: shopper_name
           - name: points
-            fidesops_meta:
+            fides_meta:
```
Comment (seanpreston) [inline src/fides/data/sample_project/sample_resources/mongo_example_test_dataset.yml:287]: 😍

**Comment by seanpreston** [comment]
Looks like that test failure is due to the ordering has change for results in `test_mongo_task`:

```
E         Full diff:
E           [
E         -  {'data_categories': ['user.unique_id'],
E         -   'field_name': 'customer_id',
E         -   'path': 'mongo_test:customer_details:customer_id'},
E            {'data_categories': ['user.date_of_birth'],
E             'field_name': 'birthday',
E             'path': 'mongo_test:customer_details:birthday'},
E            {'data_categories': ['user.childrens'],
E             'field_name': 'children',
E             'path': 'mongo_test:customer_details:children'},
E         +  {'data_categories': ['user.unique_id'],
E         +   'field_name': 'customer_id',
E         +   'path': 'mongo_test:customer_details:customer_id'},
E            {'data_categories': ['user.name'],
E             'field_name': 'emergency_contacts.name',
E             'path': 'mongo_test:customer_details:emergency_contacts.name'},
E            {'data_categories': ['user.name'],
E             'field_name': 'workplace_info.direct_reports',
E             'path': 'mongo_test:customer_details:workplace_info.direct_reports'},
E            {'data_categories': ['user.contact.phone_number'],
E             'field_name': 'emergency_contacts.phone',
E             'path': 'mongo_test:customer_details:emergency_contacts.phone'},
E            {'data_categories': ['user.gender'],
E             'field_name': 'gender',
E             'path': 'mongo_test:customer_details:gender'},
E            {'data_categories': ['user.job_title'],
E             'field_name': 'workplace_info.position',
E             'path': 'mongo_test:customer_details:workplace_info.position'},
E           ]
```

**Review by pattisdr — COMMENTED** [review]
Looks good just that failing test Sean is talking about

**Comment by NevilleS** [comment]
Ah, blargh, OK. Sorted the results and pushed a commit up... meanwhile will actually try to run this test locally 😆

**Comment by NevilleS** [comment]
OK, integration tests passed this time! The failing ones are the SaaS and AWS tests which we know have issues, so I'm mergin

**Pull request merged**